src/PDALUtils.cpp

```diff
@@ -244,15 +244,8 @@ class ArbiterInStream : public std::ifstream
 */
 std::ostream *createFile(const std::string& path, bool asBinary)
 {
-    bool remote(false);
-
-    // If we error, assume that we're local.
-    try
-    {
-        remote = arbiter::Arbiter().isRemote(path);
-    }
-    catch (arbiter::ArbiterError)
-    {}
+    arbiter::Arbiter a;
+    const bool remote(a.hasDriver(path) && a.isRemote(path));
```

Comment:
If the point of doing things this way is to avoid the exception, then it seems like the better solution is to make a single call that does that.